install/operator/deploy/crds/syndesis_v1alpha1_syndesis_crd.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: syndeses.syndesis.io
+spec:
+  group: syndesis.io
+  names:
+    kind: Syndesis
+    listKind: SyndesisList
+    plural: syndeses
+    singular: syndesis
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            components:
+              properties:
+                db:
+                  properties:
+                    database:
+                      type: string
```

Comment:
Is the operator reconciling after this fields are changed? In the current state the operator ignores the CR (I think it is only used for komodo bits)

Reply:
Nope. Nothing in that area changed.